src/lib/rule-finder.js

```diff
@@ -72,13 +72,17 @@ function _getPluginRules(config) {
 function _getAllAvailableRules(pluginRules) {
   var allRules = fs
     .readdirSync('./node_modules/eslint/lib/rules')
-    .map(function removeJsFromFilename(filename) {
-      return filename.replace(/\.js$/, '')
-    })
+  var filteredAllRules = []
+  allRules.forEach(function filterAllRules(filename) {
+    /* istanbul ignore next */
+    if (filename.substr(-3) === '.js') {
```

Comment:
this should probably use `.slice`, since `.substr` is either nonstandard or deprecated